frontend/ui/widget/checkbutton.lua

```diff
@@ -54,7 +55,7 @@ function CheckButton:initCheckButton(checked)
     self._textwidget = TextBoxWidget:new{
         text = self.text,
         face = self.face,
-        width = self.max_width - self._checkmark.dimen.w,
+        width = (self.width or self.parent._input_widget.width) - self._checkmark.dimen.w,
```

Comment:
Might want to add more check around here.
When no `self.width`, if no `self.parent`, or no `self.parent;_input_widget`, it will crash with "attempt to index a nil value".

Reply:
Sure. The coder must take care of the correct call of a widget.

Reply:
Comments added.